annotation-user/src/test/java/annotation/ProcessorTest.java

```diff
@@ -0,0 +1,25 @@
+package annotation;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ProcessorTest {
+    @Test
+    public void testUnique() throws Exception {
+        Object unique = new CreatorImpl().create("unique");
+        assertEquals(Unique.class, unique.getClass());
+    }
+    @Test
+    public void testDefault() throws Exception{
+        Object unique = new CreatorImpl().create("unique");
+        assertEquals(Unique.class, unique.getClass());
+    }
+    @Test
+    public void testNull() throws Exception{
+        Object nullObject = new CreatorImpl().create("null");
```

Comment:
Exception should be thrown in this case

Reply:
I see exception is thrown now. you don't need assertion and variable as we just expect Exception